### ISSUE TITLE
Manipulating time from console does not work

### ISSUE THREAD

**Issue by triagonal** [EssentialsX/Essentials#3621]
### Information

**Full output of `/ess version`**:

```
[23:41:36 INFO]: Server version: 1.16.1-R0.1-SNAPSHOT git-Paper-138 (MC: 1.16.1)
[23:41:36 INFO]: EssentialsX version: 2.18.1.0
[23:41:36 INFO]: Vault version: 1.7.3-b131
```

**Server log**: Relevant error: https://gist.github.com/triagonal/b6f8af2223bc7b5091d1ad93b44db991

**EssentialsX config**: Default config, no changes made.

### Details

**Description**  
Attempting to use any time manipulation commands (`/time set`, `/time add`) from console results in an error. Using these same commands ingame works as intended.

**Steps to reproduce**  
1. Run any valid time set or add command in console, e.g. `/time set day world`
2. Error occurs.

**Expected behavior**  
No error occurs and the time is changed.

**Screenshots**  
N/A

Originally reported on MOSS discord by user `𝚂𝚊𝚑𝚛𝚞𝚕*#8074`

**Comment by JRoy** [comment]
This poor, poor, command :(

**Comment by SlimeDog** [comment]
Confirmed fixed: EssentialsX 2.18.1.9
Thanks.
